rust/agama-dbus-server/src/network/model.rs

```diff
@@ -480,6 +550,47 @@ pub struct LoopbackConnection {
 }
 
 #[derive(Debug, Default, PartialEq, Clone)]
+pub struct BondConnection {
+    pub base: BaseConnection,
+    pub bond: BondConfig,
+}
+
+#[derive(Debug, Default, Clone, PartialEq)]
+pub struct BondOptions(pub HashMap<String, String>);
+
+impl TryFrom<String> for BondOptions {
+    type Error = NetworkStateError;
+
+    fn try_from(value: String) -> Result<Self, Self::Error> {
+        let mut options = HashMap::new();
+
+        for opt in value.split_whitespace() {
+            let opt_word: Vec<&str> = opt.trim().split('=').collect();
```

Comment:
You might be interested in [split_once](https://doc.rust-lang.org/std/primitive.str.html#method.split_once). Anyway, what does happen if there is no `=`?

Reply:
I'm assuming there is always a `=`, but we could be more robust.